bdaiinstitute/spot_ros2
# [PN-97] Implement Spot's Robot State Publisher in C++

This PR adds a new C++-based Spot state publisher node, which replaces the previous functionality in the Spot Python driver.

The default `spot_driver.launch.py` launch file has been updated to use the C++ publisher

### How to Test

Launch the Spot driver through its launch file, and observe that the topics containing info about Spot's state are being published.
